unit-tests/py/rspy/acroname.py

```diff
@@ -41,347 +41,343 @@ def usage():
-    volt = float(micro_volt.value) / 10.0 ** 6
-    amps = float(micro_curr.value) / 10.0 ** 6
-    #
-    return volt * amps
-
-
-def get_port_from_usb( first_usb_index, second_usb_index ):
-    """
-    Based on last two USB location index, provide the port number
-    """
-    acroname_port_usb_map = {(4, 4): 0,
-                             (4, 3): 1,
-                             (4, 2): 2,
-                             (4, 1): 3,
-                             (3, 4): 4,
-                             (3, 3): 5,
-                             (3, 2): 6,
-                             (3, 1): 7,
-                             }
-    return acroname_port_usb_map[(first_usb_index, second_usb_index)]
+        time.sleep( timeout )
+        #
+        result = self.enable_ports( portlist ) and result
+        #
+        return result
+
+
+    def set_ports_usb2(self,  portlist = None, timeout = 100e-3 ):
+        """
+        Set USB ports to USB2
+        """
+        if portlist is None:
+            portlist = self.ports()
+        #
+        self.recycle_ports( portlist, timeout = timeout )
+        #
+        for port in portlist:
+            self.hub.usb.setSuperSpeedDataEnable( port )
+            self.hub.usb.setHiSpeedDataEnable( port )
+            self.hub.usb.setSuperSpeedDataDisable( port )
+
+
+    def set_ports_usb3(self,  portlist = None, timeout = 100e-3 ):
+        """
+        Set USB ports to support USB3
+        """
+        if portlist is None:
+            portlist = self.ports()
+        #
+        self.recycle_ports( portlist, timeout = timeout )
+        #
+        for port in portlist:
+            self.hub.usb.setSuperSpeedDataEnable( port )
+            self.hub.usb.setHiSpeedDataEnable( port )
+            self.hub.usb.setHiSpeedDataDisable( port )
+
+
+    def _port_power(self,  port ):
+        """
+        """
+        if port < 0 or port > 7:
+            raise ValueError("port number can be only within 0 and 7 (inclusive)")
+        #
+        micro_volt = self.hub.usb.getPortVoltage( port )
+        micro_curr = self.hub.usb.getPortCurrent( port )
+        volt = float(micro_volt.value) / 10.0 ** 6
+        amps = float(micro_curr.value) / 10.0 ** 6
+        #
+        return volt * amps
+
+
+    def get_port_from_usb(self, first_usb_index, second_usb_index ):
+        """
+        Based on last two USB location index, provide the port number
+        """
+        acroname_port_usb_map = {(4, 4): 0,
+                                 (4, 3): 1,
+                                 (4, 2): 2,
+                                 (4, 1): 3,
+                                 (3, 4): 4,
+                                 (3, 3): 5,
+                                 (3, 2): 6,
+                                 (3, 1): 7,
+                                 }
+        return acroname_port_usb_map[(first_usb_index, second_usb_index)]
 
 
 if __name__ == '__main__':
+    device = Acroname()
```

Comment:
I say lets call it `acroname`  (lower case) and not device as we call our cameras `devices`